.github/workflows/ci.yml

```
@@ -5,7 +5,7 @@ on:
     branches: 
         - main
   pull_request:
-    branches: [ * ]
+    branches:
```

actionlint (1.7.7)

string should not be empty

(syntax-check)